packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -29,10 +64,19 @@ type State = {|
 
 export default class LabeledTextField extends React.Component<Props, State> {
     static defaultProps: DefaultProps = {
+        type: "text",
         disabled: false,
     };
 
+    constructor(props: Props) {
+        super(props);
+        if (props.initialValue) {
+            this.state.value = props.initialValue;
+        }
```

Comment:
Would it matter if this were just `this.state.value = props.initialValue` without the check? Seems like the effect would be the same.

If it needs to retain a default, then what about.

`this.state.value = props.initialValue || "";`

Reply:
Good suggestion! I could also just make initialValue a default prop and set it to `""` by default.